move.go

```diff
@@ -38,6 +38,16 @@ type Move struct {
 	Accuracy int
 }
 
+// Creates a new Pokemon move given its move ID
+func NewMove(id int) Move {
```

Comment:
(nit) Implies that the user is creating a brand new move, one that is not in the pokedex data.

`GetMove` already exists in pokedex. pokedex is currently being treated as a database kinda, where the functions inside are specifically ones to retrieve information from that database. It'll be easier to make it a sub-package later, if we want to go that route.

Reply:
I interpreted it to be like the way the `new` keyword is used in say Java, to imply that's a constructor. Can we make functions in go that are the same as the name of a type?

Reply:
From what I've seen in Go, `NewMyType` functions are used to create initial values for custom types. For some examples, see Go's [image](https://golang.org/pkg/image/) package and [this blurb](https://golang.org/doc/effective_go.html#allocation_new) about creating new types. If `move` became an unexported type, then we can use `Move()` as the constructor.

I think the generated data should serve as a database and not as a base struct value. The idea with having constructors is that we can pass in parameters (e.g. custom PP, IVs for Pokemon, metadata for battles) and not have users concerned as much about the internal representation/implementation of `Move`/`Battle`. We just have to enforce standard (#61 likely relevant).

Reply:
> I think the generated data should serve as a database and not as a base struct value.

Yes, precisely. This function is being used to grab a move from the database, and use it as a template.
The name `NewMove` should be reserved for creating a move that is not in the database (which is not yet implemented).